pkg/etcdcli/health.go

```diff
@@ -132,7 +132,9 @@ func checkSingleMemberHealth(ctx context.Context, member *etcdserverpb.Member) h
 		}
 		hc.Healthy = true
 	} else {
-		klog.Errorf("health check for memberID (%v) failed: err(%v)", resp.Header.MemberId, err)
+		if resp.Header != nil {
```

Comment:
so we never log an error when there is an error but no response?

Reply:
this pr is primarily intended to get around the panic that we hit caused by not having a nil check here on the log. Logs themselves can be updated to whatever you like.

Reply:
sure, thanks for your help, can we backport this one as well?

https://github.com/openshift/cluster-etcd-operator/pull/922

Reply:
yeah that makes sense to me, i was wondering why resp was being used in the first place :)